### PR TITLE
rbd: log stderror when running modprobe

### PR DESCRIPTION
**Pull request by Madhu-1**
logging the error is not user-friendly and it contains the system error message. Log the stderr which is the user-friendly error message for identifying the problem.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

File: internal/rbd/rbd_attach.go [hunk 1]
```diff
@@ -199,14 +199,15 @@ func waitForPath(ctx context.Context, pool, namespace, image string, maxRetries
...
 	// check if the module is loaded or compiled in
 	_, err := os.Stat(fmt.Sprintf("/sys/module/%s", moduleNbd))
 	if os.IsNotExist(err) {
 		// try to load the module
-		_, _, err = util.ExecCommand(context.TODO(), "modprobe", moduleNbd)
+		_, stderr, err = util.ExecCommand(context.TODO(), "modprobe", moduleNbd)
 		if err != nil {
 			hasNBD = false
-			log.WarningLogMsg("rbd-nbd: nbd modprobe failed with error %v", err)
+			log.WarningLogMsg("rbd-nbd: nbd modprobe failed with error:%v, stderr:%s", err, stderr)
```
Comment (Rakshith-R) [inline internal/rbd/rbd_attach.go]:
```suggestion
			log.WarningLogMsg("rbd-nbd: nbd modprobe failed with error:%v, stderr:%q", err, stderr)
```

File: internal/rbd/rbd_util.go [hunk 2]
```diff
@@ -223,9 +224,9 @@ func GetKrbdSupportedFeatures() (string, error) {
 			return "", err
 		}
 		// try to load the module
-		_, _, err = util.ExecCommand(context.TODO(), "modprobe", rbdDefaultMounter)
+		_, stderr, err = util.ExecCommand(context.TODO(), "modprobe", rbdDefaultMounter)
 		if err != nil {
-			log.ErrorLogMsg("modprobe failed: %v", err)
+			log.ErrorLogMsg("modprobe failed:%v stderr:%s", err, stderr)
```
Comment (Rakshith-R) [inline internal/rbd/rbd_util.go]:
```suggestion
			log.ErrorLogMsg("modprobe failed:%v stderr:%q", err, stderr)
```

**Review by Rakshith-R — DISMISSED** [review]
Just some tiny changes but lgtm.

File: internal/rbd/rbd_attach.go (hunk 1 above)
Comment (nixpanic) [reply internal/rbd/rbd_attach.go]: Maybe use the following format? It removes the unneeded `with error`, and reduces one `:` to improve readability.

```go
log.WarningLogMsg("rbd-nbd: nbd modprobe failed (%v): %s", err, stderr)
```

**Review by Rakshith-R — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]

**Comment by nixpanic** [comment]
> @Madhu-1 "ci/centos/upgrade-tests-cephfs" test failed. Logs are available at [location](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/job/upgrade-tests-cephfs/4681/display/redirect) for debugging

Should be addressed with #2880

**Pull request merged**